chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -1,7 +1,15 @@
 include_recipe "ceph::default"
 include_recipe "ceph::conf"
+
 # needed because of our use of ceph_client
-include_recipe "ceph::keyring"
+ceph_env_filter = " AND ceph_config_environment:ceph-config-default"
+ceph_servers = search(:node, "roles:ceph-osd#{ceph_env_filter}") || []
+if ceph_servers.length > 0
+  include_recipe "ceph::keyring"
+else
+  Chef::Log.info("Ceph was not deployed with Crowbar yet!")
+  return
```

Comment:
Hrm. But that means that after "apply", radosgw will actually not work and we'll have to wait 15 minutes.

I think the right fix would be to not include the keyring recipe if we're a mon, as I believe this is what is creating the issue @bmwiedemann saw.